### DISCUSSION TITLE
Welcome to Snake-Game-OOP Discussions!

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [thuongtruong1009/Snake-Game-OOP#5] [Q&A] [answered]
## 👋 Welcome to github.com/thuongtruong1009 !
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thuongtruong109** [comment, marked as answer]
@thuongtruong1009 
You should fix the link of sound and image, which should get in the internal source code. Because itn't be changed  when you change on the device.
It shoud put in .('./link-directed/....');

**Reply by thuongtruong1009** [reply]
thanks you very much

**Comment by thuongtruong109** [comment]
> ## 👋 Welcome to github.com/thuongtruong1009 !
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  I can discuss and it can add me in contributing?
This is good for practice analyse the code.

**Reply by thuongtruong1009** [reply]
Thank you verry much!
